test/kubetest/k8s.go

```diff
@@ -378,9 +378,20 @@ func NewK8sWithClientset(clientset kubernetes.Interface, namespace string) *K8s
 	return r
 }
 
+func runShell(cmd string) {
```

Comment:
Can you just add

`kubectl describe nodes`  into https://github.com/networkservicemesh/networkservicemesh/blob/master/.cloudtest/executions/single-cluster-tests.yaml#L17?

Reply:
I'll remove it as it is no longer needed.